docs/content/doc/usage/backup-and-restore.en-us.md

```diff
@@ -95,3 +95,37 @@ Repository Git Hooks should be regenerated if installation method is changed (eg
 With Gitea running, and from the directory Gitea's binary is located, execute: `./gitea admin regenerate hooks`
 
 This ensures that application and configuration file paths in repository Git Hooks are consistent and applicable to the current installation. If these paths are not updated, repository `push` actions will fail.
+
+### Using Docker (`restore`)
+
+There is also no support fot a recovery command in a docker based gitea instance. The restore process contains the same steps like described in re previous section but with different paths.
+
+Example:
+
+```sh
+# open bash session in container
+docker exec --user git -it <container-id | service-id> bash
+# unzip your backup file within the container
+unzip gitea-dump-1610949662.zip
```

Comment:
I think these should be replaced by a placeholder like `xxxxx` to indicate it could be a variety of numbers.

Reply:
Thanks for helping me out 🥇